python/tests/conftest.py

```diff
@@ -153,13 +153,19 @@ def table_schema_nested() -> Schema:
         NestedField(
             field_id=11,
             name="location",
-            field_type=ListType(
-                element_id=12,
-                element_type=StructType(
+            field_type=MapType(
```

Comment:
Maybe better to not change this fixture, and create a new schema?

Reply:
reverted.